src/pivot-table/components/helpers/get-item-size-handler.ts

```diff
@@ -31,24 +29,20 @@ export const getRowHeightHandler =
   };
 
 export const getColumnWidthHandler =
-  ({
-    list,
-    isLastRow,
-    layoutService,
-    allMeasuresWidth,
-    getMeasureInfoWidth,
-  }: ColumnWidthHandlerProps): ItemSizeHandler =>
+  ({ list, isLastRow, getLeafWidth }: ColumnWidthHandlerProps): ItemSizeHandler =>
   (colIndex: number) => {
     const cell = isLastRow ? list[colIndex] : Object.values(list)[colIndex];
-    const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
+    // const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
 
-    if (colIndex === 0 && cell?.x > 0) {
-      return ((cell.leafCount + cell.x) / measureInfoCount) * allMeasuresWidth;
-    }
+    // Dunno when this is true
+    // if (colIndex === 0 && cell?.x > 0) {
```

Comment:
Never understood when this is true?

Reply:
It's a bit of a special case but if you are on a different page then the first. Scroll down a bit and expand a node. The "first row" does not exist yet as data has only been fetched for the rows that are visible to the user.

Reply:
FYI this scenario should still remain relevant

Reply:
Yeah I figured I would look into it.

Reply:
scoped out of this PR, see description

Reply:
Doing some testing I found a common case where the scroll position is lost when expanding a node. Just scroll a bit to a node and expand it. Doing that seem to cause the scroll position to get lost as it's no longer positioned at the expanded node.

This scenario works just fine on main. I'll do some more digging on this.

Reply:
Seem to only happen when doing it on the top grid

Reply:
Do you think I need to fix it in this one? I can look at it the first thing in the next PR

Reply:
Most likely not, sure yet. Need to figure out when and why it happens. If it's a likely scenario to trigger or not

Reply:
Something is certianly broken as it fails to render cells in this case. I just scroll right (more than 50 columns) and expanded a node

![Skärmavbild 2023-09-22 kl  10 26 57](https://github.com/qlik-oss/sn-pivot-table/assets/16608020/8166a21e-ceaf-40cc-b807-5189e4c0c0f6)

Reply:
> Doing some testing I found a common case where the scroll position is lost when expanding a node. Just scroll a bit to a node and expand it. Doing that seem to cause the scroll position to get lost as it's no longer positioned at the expanded node.

This seem to be because the width of the columns changes as the data changes when you expand/collapse. It's on main as well. So that should be fine to deliver with.